app/(root)/posts/page.tsx

```diff
@@ -1,80 +1,35 @@
-'use client';
+"use client";
 
-import React from 'react';
-// import Link from 'next/link';
-import { useState } from 'react';
-import { updateUser, deleteUser } from '@/lib/actions/user.actions';
+import React from "react";
+import Link from "next/link";
+// import { deleteUser } from "@/lib/actions/user.actions";
 
-const Posts = (user: any) => {
-  console.log('USER', user);
-  const [data, setData] = useState({
-    id: user && user.id,
-    name: user && user.name,
-    email: user && user.email,
-  });
+const Posts = () => {
+  // const handleDelete = () => {
```

Comment:
Keeping this commented out code here for now incase I need to test the 'Delete User' functionality later on